Starboard: Observed a panic: "invalid memory address or nil pointer dereference"

**Describe the bug**
Starboard operator crashlooping in service cluster.

```
{"level":"info","ts":1629790324.78894,"logger":"controller-runtime.manager.controller.job","msg":"Starting workers","reconciler group":"batch","reconciler kind":"Job","worker count":1}
E0824 07:32:04.833468       1 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
goroutine 395 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x15e4da0, 0x22e4a50)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/runtime/runtime.go:74 +0xa6
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/runtime/runtime.go:48 +0x86
panic(0x15e4da0, 0x22e4a50)
	/opt/hostedtoolcache/go/1.16.3/x64/src/runtime/panic.go:965 +0x1b9
github.com/aquasecurity/starboard/pkg/kube.(*logsReader).GetLogsByJobAndContainerName(0xc000342e10, 0x19868d8, 0xc0011db500, 0xc0013a8000, 0xc001362b98, 0x4, 0xe, 0xc000607099, 0x3, 0xc0006070b0)
	/home/runner/work/starboard/starboard/pkg/kube/logs.go:35 +0x1e5
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).processCompleteScanJob(0xc000143440, 0x19868d8, 0xc0011db500, 0xc0013a8000, 0xa, 0xc000682450)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:375 +0xb28
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).reconcileJobs.func1(0x19868d8, 0xc0011db500, 0xc000607080, 0xa, 0xc000682450, 0x23, 0x16ad340, 0xc0011db500, 0x15e33c0, 0x23351c8)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:323 +0x31b
sigs.k8s.io/controller-runtime/pkg/reconcile.Func.Reconcile(0xc000343650, 0x19868d8, 0xc0011db500, 0xc000607080, 0xa, 0xc000682450, 0x23, 0xc0011db500, 0xc000030000, 0x166e740, ...)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/reconcile/reconcile.go:102 +0x6f
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0002748c0, 0x1986830, 0xc0001700c0, 0x1637940, 0xc000b394c0)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:263 +0x30d
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0002748c0, 0x1986830, 0xc0001700c0, 0x0)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:235 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.1(0x1986830, 0xc0001700c0)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:198 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc00008a750)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc002295f50, 0x1958280, 0xc0011db4a0, 0xc000170001, 0xc000148240)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:156 +0x9b
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00008a750, 0x3b9aca00, 0x0, 0xc000936a01, 0xc000148240)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x1986830, 0xc0001700c0, 0xc001189210, 0x3b9aca00, 0x0, 0x183ec01)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x1986830, 0xc0001700c0, 0xc001189210, 0x3b9aca00)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:195 +0x497
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x13a2505]

goroutine 395 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/runtime/runtime.go:55 +0x109
panic(0x15e4da0, 0x22e4a50)
	/opt/hostedtoolcache/go/1.16.3/x64/src/runtime/panic.go:965 +0x1b9
github.com/aquasecurity/starboard/pkg/kube.(*logsReader).GetLogsByJobAndContainerName(0xc000342e10, 0x19868d8, 0xc0011db500, 0xc0013a8000, 0xc001362b98, 0x4, 0xe, 0xc000607099, 0x3, 0xc0006070b0)
	/home/runner/work/starboard/starboard/pkg/kube/logs.go:35 +0x1e5
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).processCompleteScanJob(0xc000143440, 0x19868d8, 0xc0011db500, 0xc0013a8000, 0xa, 0xc000682450)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:375 +0xb28
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).reconcileJobs.func1(0x19868d8, 0xc0011db500, 0xc000607080, 0xa, 0xc000682450, 0x23, 0x16ad340, 0xc0011db500, 0x15e33c0, 0x23351c8)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:323 +0x31b
sigs.k8s.io/controller-runtime/pkg/reconcile.Func.Reconcile(0xc000343650, 0x19868d8, 0xc0011db500, 0xc000607080, 0xa, 0xc000682450, 0x23, 0xc0011db500, 0xc000030000, 0x166e740, ...)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/reconcile/reconcile.go:102 +0x6f
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0002748c0, 0x1986830, 0xc0001700c0, 0x1637940, 0xc000b394c0)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:263 +0x30d
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0002748c0, 0x1986830, 0xc0001700c0, 0x0)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:235 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.1(0x1986830, 0xc0001700c0)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:198 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc00008a750)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc002295f50, 0x1958280, 0xc0011db4a0, 0xc000170001, 0xc000148240)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:156 +0x9b
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00008a750, 0x3b9aca00, 0x0, 0xc000936a01, 0xc000148240)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x1986830, 0xc0001700c0, 0xc001189210, 0x3b9aca00, 0x0, 0x183ec01)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x1986830, 0xc0001700c0, 0xc001189210, 0x3b9aca00)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:195 +0x497
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install starboard operator
2. Maybe notice error

**Expected behavior**

**Screenshots**

**Version (add all relevant versions):**
v0.16.0

**Additional context**

Full Log
```
❯ sokctl -n monitoring logs starboard-operator-845c46db7-nsw72
{"level":"info","ts":1629790502.9144459,"logger":"main","msg":"Starting operator","buildInfo":{"Version":"0.10.1","Commit":"21e893cb41c75144f9877735303bbf783c3f80bc","Date":"2021-04-21T19:48:38Z","Executable":""}}
{"level":"info","ts":1629790502.9151711,"logger":"operator","msg":"Resolved install mode","install mode":"AllNamespaces","operator namespace":"monitoring","target namespaces":[]}
{"level":"info","ts":1629790502.9158082,"logger":"operator","msg":"Watching all namespaces"}
I0824 07:35:03.969914       1 request.go:645] Throttling request took 1.034564169s, request: GET:https://10.96.0.1:443/apis/monitoring.coreos.com/v1alpha1?timeout=32s
{"level":"info","ts":1629790504.1242743,"logger":"controller-runtime.metrics","msg":"metrics server is starting to listen","addr":":8080"}
{"level":"info","ts":1629790504.2571225,"logger":"operator","msg":"Starting controllers manager"}
{"level":"info","ts":1629790504.2574904,"logger":"controller-runtime.manager","msg":"starting metrics server","path":"/metrics"}
{"level":"info","ts":1629790504.2577283,"logger":"controller-runtime.manager.controller.pod","msg":"Starting EventSource","reconciler group":"","reconciler kind":"Pod","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.258035,"logger":"controller-runtime.manager.controller.replicaset","msg":"Starting EventSource","reconciler group":"apps","reconciler kind":"ReplicaSet","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.2582946,"logger":"controller-runtime.manager.controller.replicationcontroller","msg":"Starting EventSource","reconciler group":"","reconciler kind":"ReplicationController","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.2584515,"logger":"controller-runtime.manager.controller.statefulset","msg":"Starting EventSource","reconciler group":"apps","reconciler kind":"StatefulSet","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.258666,"logger":"controller-runtime.manager.controller.daemonset","msg":"Starting EventSource","reconciler group":"apps","reconciler kind":"DaemonSet","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.258878,"logger":"controller-runtime.manager.controller.cronjob","msg":"Starting EventSource","reconciler group":"batch","reconciler kind":"CronJob","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.2590027,"logger":"controller-runtime.manager.controller.job","msg":"Starting EventSource","reconciler group":"batch","reconciler kind":"Job","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.2592008,"logger":"controller-runtime.manager.controller.job","msg":"Starting EventSource","reconciler group":"batch","reconciler kind":"Job","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.358832,"logger":"controller-runtime.manager.controller.replicationcontroller","msg":"Starting EventSource","reconciler group":"","reconciler kind":"ReplicationController","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.3590984,"logger":"controller-runtime.manager.controller.pod","msg":"Starting EventSource","reconciler group":"","reconciler kind":"Pod","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.359313,"logger":"controller-runtime.manager.controller.replicaset","msg":"Starting EventSource","reconciler group":"apps","reconciler kind":"ReplicaSet","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.359531,"logger":"controller-runtime.manager.controller.job","msg":"Starting Controller","reconciler group":"batch","reconciler kind":"Job"}
{"level":"info","ts":1629790504.3588345,"logger":"controller-runtime.manager.controller.statefulset","msg":"Starting EventSource","reconciler group":"apps","reconciler kind":"StatefulSet","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.3595598,"logger":"controller-runtime.manager.controller.daemonset","msg":"Starting EventSource","reconciler group":"apps","reconciler kind":"DaemonSet","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.3595915,"logger":"controller-runtime.manager.controller.cronjob","msg":"Starting EventSource","reconciler group":"batch","reconciler kind":"CronJob","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.3596883,"logger":"controller-runtime.manager.controller.job","msg":"Starting EventSource","reconciler group":"batch","reconciler kind":"Job","source":"kind source: /, Kind="}
{"level":"info","ts":1629790504.5608222,"logger":"controller-runtime.manager.controller.cronjob","msg":"Starting Controller","reconciler group":"batch","reconciler kind":"CronJob"}
{"level":"info","ts":1629790504.5609388,"logger":"controller-runtime.manager.controller.cronjob","msg":"Starting workers","reconciler group":"batch","reconciler kind":"CronJob","worker count":1}
{"level":"info","ts":1629790504.5612557,"logger":"controller-runtime.manager.controller.replicationcontroller","msg":"Starting Controller","reconciler group":"","reconciler kind":"ReplicationController"}
{"level":"info","ts":1629790504.5613558,"logger":"controller-runtime.manager.controller.replicationcontroller","msg":"Starting workers","reconciler group":"","reconciler kind":"ReplicationController","worker count":1}
{"level":"info","ts":1629790504.5615878,"logger":"controller-runtime.manager.controller.replicaset","msg":"Starting Controller","reconciler group":"apps","reconciler kind":"ReplicaSet"}
{"level":"info","ts":1629790504.5616016,"logger":"controller-runtime.manager.controller.replicaset","msg":"Starting workers","reconciler group":"apps","reconciler kind":"ReplicaSet","worker count":1}
{"level":"info","ts":1629790504.5655894,"logger":"controller-runtime.manager.controller.job","msg":"Starting Controller","reconciler group":"batch","reconciler kind":"Job"}
{"level":"info","ts":1629790504.565684,"logger":"controller-runtime.manager.controller.job","msg":"Starting workers","reconciler group":"batch","reconciler kind":"Job","worker count":1}
{"level":"info","ts":1629790504.5701675,"logger":"controller-runtime.manager.controller.daemonset","msg":"Starting Controller","reconciler group":"apps","reconciler kind":"DaemonSet"}
{"level":"info","ts":1629790504.5703683,"logger":"controller-runtime.manager.controller.daemonset","msg":"Starting workers","reconciler group":"apps","reconciler kind":"DaemonSet","worker count":1}
{"level":"info","ts":1629790504.5723705,"logger":"controller-runtime.manager.controller.pod","msg":"Starting Controller","reconciler group":"","reconciler kind":"Pod"}
{"level":"info","ts":1629790504.5723934,"logger":"controller-runtime.manager.controller.pod","msg":"Starting workers","reconciler group":"","reconciler kind":"Pod","worker count":1}
{"level":"info","ts":1629790504.593973,"logger":"controller-runtime.manager.controller.statefulset","msg":"Starting Controller","reconciler group":"apps","reconciler kind":"StatefulSet"}
{"level":"info","ts":1629790504.5940654,"logger":"controller-runtime.manager.controller.statefulset","msg":"Starting workers","reconciler group":"apps","reconciler kind":"StatefulSet","worker count":1}
{"level":"info","ts":1629790504.6001377,"logger":"controller-runtime.manager.controller.job","msg":"Starting workers","reconciler group":"batch","reconciler kind":"Job","worker count":1}
E0824 07:35:04.622821       1 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
goroutine 402 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x15e4da0, 0x22e4a50)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/runtime/runtime.go:74 +0xa6
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/runtime/runtime.go:48 +0x86
panic(0x15e4da0, 0x22e4a50)
	/opt/hostedtoolcache/go/1.16.3/x64/src/runtime/panic.go:965 +0x1b9
github.com/aquasecurity/starboard/pkg/kube.(*logsReader).GetLogsByJobAndContainerName(0xc00003bb40, 0x19868d8, 0xc001ca4ff0, 0xc001aded80, 0xc000b4fb30, 0x4, 0xe, 0xc00026df29, 0x3, 0xc00026df40)
	/home/runner/work/starboard/starboard/pkg/kube/logs.go:35 +0x1e5
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).processCompleteScanJob(0xc0001417a0, 0x19868d8, 0xc001ca4ff0, 0xc001aded80, 0xa, 0xc000041c50)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:375 +0xb28
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).reconcileJobs.func1(0x19868d8, 0xc001ca4ff0, 0xc00026df10, 0xa, 0xc000041c50, 0x22, 0x16ad340, 0xc001ca4ff0, 0x15e33c0, 0x23351c8)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:323 +0x31b
sigs.k8s.io/controller-runtime/pkg/reconcile.Func.Reconcile(0xc00009b3f0, 0x19868d8, 0xc001ca4ff0, 0xc00026df10, 0xa, 0xc000041c50, 0x22, 0xc001ca4ff0, 0xc000030000, 0x166e740, ...)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/reconcile/reconcile.go:102 +0x6f
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0002f25a0, 0x1986830, 0xc000568980, 0x1637940, 0xc000ac8b80)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:263 +0x30d
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0002f25a0, 0x1986830, 0xc000568980, 0xc00055de00)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:235 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.1(0x1986830, 0xc000568980)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:198 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc00055df50)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc001587f50, 0x1958280, 0xc001ca4f90, 0xc000568901, 0xc00007e1e0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:156 +0x9b
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00055df50, 0x3b9aca00, 0x0, 0x1271e01, 0xc00007e1e0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x1986830, 0xc000568980, 0xc0006331a0, 0x3b9aca00, 0x0, 0x1)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x1986830, 0xc000568980, 0xc0006331a0, 0x3b9aca00)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:195 +0x497
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x13a2505]

goroutine 402 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/runtime/runtime.go:55 +0x109
panic(0x15e4da0, 0x22e4a50)
	/opt/hostedtoolcache/go/1.16.3/x64/src/runtime/panic.go:965 +0x1b9
github.com/aquasecurity/starboard/pkg/kube.(*logsReader).GetLogsByJobAndContainerName(0xc00003bb40, 0x19868d8, 0xc001ca4ff0, 0xc001aded80, 0xc000b4fb30, 0x4, 0xe, 0xc00026df29, 0x3, 0xc00026df40)
	/home/runner/work/starboard/starboard/pkg/kube/logs.go:35 +0x1e5
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).processCompleteScanJob(0xc0001417a0, 0x19868d8, 0xc001ca4ff0, 0xc001aded80, 0xa, 0xc000041c50)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:375 +0xb28
github.com/aquasecurity/starboard/pkg/operator/controller.(*VulnerabilityReportReconciler).reconcileJobs.func1(0x19868d8, 0xc001ca4ff0, 0xc00026df10, 0xa, 0xc000041c50, 0x22, 0x16ad340, 0xc001ca4ff0, 0x15e33c0, 0x23351c8)
	/home/runner/work/starboard/starboard/pkg/operator/controller/vulnerabilityreport.go:323 +0x31b
sigs.k8s.io/controller-runtime/pkg/reconcile.Func.Reconcile(0xc00009b3f0, 0x19868d8, 0xc001ca4ff0, 0xc00026df10, 0xa, 0xc000041c50, 0x22, 0xc001ca4ff0, 0xc000030000, 0x166e740, ...)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/reconcile/reconcile.go:102 +0x6f
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0002f25a0, 0x1986830, 0xc000568980, 0x1637940, 0xc000ac8b80)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:263 +0x30d
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0002f25a0, 0x1986830, 0xc000568980, 0xc00055de00)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:235 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.1(0x1986830, 0xc000568980)
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:198 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc00055df50)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc001587f50, 0x1958280, 0xc001ca4f90, 0xc000568901, 0xc00007e1e0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:156 +0x9b
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00055df50, 0x3b9aca00, 0x0, 0x1271e01, 0xc00007e1e0)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x1986830, 0xc000568980, 0xc0006331a0, 0x3b9aca00, 0x0, 0x1)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x1986830, 0xc000568980, 0xc0006331a0, 0x3b9aca00)
	/home/runner/go/pkg/mod/k8s.io/apimachinery@v0.19.2/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.7.2/pkg/internal/controller/controller.go:195 +0x497
```

--- a/WIP-CHANGELOG.md
+++ b/WIP-CHANGELOG.md
@@ -10,6 +10,7 @@
 # Updated
 
 - Updated influxdb chart 4.8.12 to 4.8.15
+- Updated starboard-operator chart from v0.5.1 (app v0.10.1) to v0.7.0 (app v0.12.0), this introduces a PSP RBAC as a subchart since the Trivy scanners were unable to run.
 
 ### Changed
 

--- a/helmfile/50-applications.yaml
+++ b/helmfile/50-applications.yaml
@@ -123,7 +123,7 @@ releases:
   labels:
     app: starboard-operator
   chart: ./upstream/starboard-operator
-  version: 0.5.1
+  version: 0.7.0
   missingFileHandler: Error
   values:
   - values/starboard-operator.yaml.gotmpl

--- a/helmfile/upstream/README.md
+++ b/helmfile/upstream/README.md
@@ -5,3 +5,7 @@ helm repo add falcosecurity https://falcosecurity.github.io/charts
 helm repo up
 helm fetch falcosecurity/falco --version 1.5.2 --untar
 ```
+
+# Note
+
+The Starboard Operator currently uses a subchart containing a PSP to allow the Trivy scanners to run and the RBAC to use it. Keep it until it is supported upstream.

--- a/helmfile/upstream/starboard-operator/Chart.yaml
+++ b/helmfile/upstream/starboard-operator/Chart.yaml
@@ -6,12 +6,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.5.1
+version: 0.7.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 0.10.1
+appVersion: 0.12.0
 
 # kubeVersion: A SemVer range of compatible Kubernetes versions (optional)
 
@@ -29,3 +29,7 @@ sources:
 # icon: A URL to an SVG or PNG image to be used as an icon (optional).
 # annotations:
 #   example: A list of annotations keyed by name (optional).
+
+dependencies:
+  - name: starboard-operator-psp-rbac
+    version: 0.1.0

--- a/helmfile/upstream/starboard-operator/charts/starboard-operator-psp-rbac/.helmignore
+++ b/helmfile/upstream/starboard-operator/charts/starboard-operator-psp-rbac/.helmignore
@@ -1,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*.orig
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/

--- a/helmfile/upstream/starboard-operator/charts/starboard-operator-psp-rbac/Chart.yaml
+++ b/helmfile/upstream/starboard-operator/charts/starboard-operator-psp-rbac/Chart.yaml
@@ -1,0 +1,5 @@
+apiVersion: v2
+appVersion: "1.0"
+description: Starboard Operator PSP RBAC
+name: starboard-operator-psp-rbac
+version: 0.1.0

--- a/helmfile/upstream/starboard-operator/charts/starboard-operator-psp-rbac/templates/starboard-operator-psp-rbac.yaml
+++ b/helmfile/upstream/starboard-operator/charts/starboard-operator-psp-rbac/templates/starboard-operator-psp-rbac.yaml
@@ -1,0 +1,63 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: 00-starboard-operator
+spec:
+  allowPrivilegeEscalation: false
+  fsGroup:
+    ranges:
+    - max: 65535
+      min: 1
+    rule: MustRunAs
+  requiredDropCapabilities:
+  - ALL
+  runAsGroup:
+    ranges:
+    - max: 65535
+      min: 1
+    rule: MustRunAs
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    ranges:
+    - max: 65535
+      min: 1
+    rule: MustRunAs
+  volumes:
+  - configMap
+  - emptyDir
+  - projected
+  - secret
+  - downwardAPI
+  - persistentVolumeClaim
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: starboard-operator-psp
+  namespace: monitoring
+rules:
+- apiGroups:
+  - extensions
+  resourceNames:
+  - 00-starboard-operator
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: starboard-operator-psp
+  namespace: monitoring
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: starboard-operator-psp
+subjects:
+- kind: ServiceAccount
+  name: starboard-operator
+  namespace: monitoring

--- a/helmfile/upstream/starboard-operator/crds/clusterconfigauditreports.crd.yaml
+++ b/helmfile/upstream/starboard-operator/crds/clusterconfigauditreports.crd.yaml
@@ -1,0 +1,50 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterconfigauditreports.aquasecurity.github.io
+  labels:
+    app.kubernetes.io/managed-by: starboard
+spec:
+  group: aquasecurity.github.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+        - jsonPath: .report.scanner.name
+          type: string
+          name: Scanner
+          description: The name of the config audit scanner
+        - jsonPath: .metadata.creationTimestamp
+          type: date
+          name: Age
+          description: The age of the report
+        - jsonPath: .report.summary.dangerCount
+          type: integer
+          name: Danger
+          priority: 1
+          description: The number of checks that failed with Danger status
+        - jsonPath: .report.summary.warningCount
+          type: integer
+          name: Warning
+          priority: 1
+          description: The number of checks that failed with Warning status
+        - jsonPath: .report.summary.passCount
+          type: integer
+          name: Pass
+          priority: 1
+          description: The number of checks that passed
+      schema:
+        openAPIV3Schema:
+          x-kubernetes-preserve-unknown-fields: true
+          type: object
+  scope: Cluster
+  names:
+    singular: clusterconfigauditreport
+    plural: clusterconfigauditreports
+    kind: ClusterConfigAuditReport
+    listKind: ClusterConfigAuditReportList
+    categories:
+      - all
+    shortNames:
+      - clusterconfigaudit

--- a/helmfile/upstream/starboard-operator/crds/vulnerabilityreports.crd.yaml
+++ b/helmfile/upstream/starboard-operator/crds/vulnerabilityreports.crd.yaml
@@ -31,11 +31,15 @@ spec:
             report:
               type: object
               required:
+                - updateTimestamp
                 - scanner
                 - artifact
                 - summary
                 - vulnerabilities
               properties:
+                updateTimestamp:
+                  type: string
+                  format: date-time
                 scanner:
                   type: object
                   required:

--- a/helmfile/upstream/starboard-operator/templates/config.yaml
+++ b/helmfile/upstream/starboard-operator/templates/config.yaml
@@ -6,33 +6,85 @@ metadata:
   labels:
     {{- include "starboard-operator.labels" . | nindent 4 }}
 data:
+  scanJob.tolerations: {{ .Values.starboard.scanJobTolerations | toJson | quote }}
+  scanJob.annotations: {{ .Values.starboard.scanJobAnnotations | quote }}
   {{- if .Values.operator.vulnerabilityScannerEnabled }}
-  vulnerabilityReports.scanner: {{ .Values.operator.vulnerabilityReportsPlugin | quote }}
-  {{- if eq .Values.operator.vulnerabilityReportsPlugin "Trivy" }}
-  trivy.imageRef: "{{ .Values.trivy.imageRef }}"
-  trivy.mode: "{{ .Values.trivy.mode }}"
-  {{- if eq .Values.trivy.mode "ClientServer" }}
-  trivy.serverURL: "{{ .Values.trivy.serverURL }}"
-  {{- end }}
-  trivy.httpProxy: "{{ .Values.trivy.httpProxy }}"
-  trivy.httpsProxy: "{{ .Values.trivy.httpsProxy }}"
-  trivy.noProxy: "{{ .Values.trivy.noProxy }}"
-  trivy.severity: "{{ .Values.trivy.severity }}"
-  {{- end }}
+  vulnerabilityReports.scanner: {{ .Values.starboard.vulnerabilityReportsPlugin | quote }}
   {{- end }}
   {{- if .Values.operator.kubernetesBenchmarkEnabled }}
-  kube-bench.imageRef: "{{ .Values.kubeBench.imageRef }}"
+  kube-bench.imageRef: {{ required ".Values.kubeBench.imageRef is required" .Values.kubeBench.imageRef | quote }}
   {{- end }}
   {{- if .Values.operator.configAuditScannerEnabled }}
-  configAuditReports.scanner: {{ .Values.operator.configAuditReportsPlugin | quote }}
-  {{- if eq .Values.operator.configAuditReportsPlugin "Polaris" }}
-  polaris.imageRef: "{{ .Values.polaris.imageRef }}"
+  configAuditReports.scanner: {{ .Values.starboard.configAuditReportsPlugin | quote }}
   {{- end }}
-  {{- if eq .Values.operator.configAuditReportsPlugin "Conftest" }}
-  conftest.imageRef: {{ .Values.conftest.imageRef | quote }}
+{{- if eq .Values.starboard.vulnerabilityReportsPlugin "Trivy" }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: starboard-trivy-config
+  labels:
+    {{- include "starboard-operator.labels" . | nindent 4 }}
+data:
+  trivy.imageRef: {{ required ".Values.trivy.imageRef is required" .Values.trivy.imageRef | quote }}
+  trivy.mode: {{ .Values.trivy.mode | quote }}
+  {{- if .Values.trivy.httpProxy }}
+  trivy.httpProxy: {{ .Values.trivy.httpProxy | quote }}
+  {{- end }}
+  {{- if .Values.trivy.httpsProxy }}
+  trivy.httpsProxy: {{ .Values.trivy.httpsProxy | quote }}
+  {{- end }}
+  {{- if .Values.trivy.noProxy }}
+  trivy.noProxy: {{ .Values.trivy.noProxy | quote }}
+  {{- end }}
+  trivy.severity: {{ .Values.trivy.severity | quote }}
+  {{- if .Values.trivy.ignoreUnfixed }}
+  trivy.ignoreUnfixed: {{ .Values.trivy.ignoreUnfixed | quote }}
+  {{- end }}
+  {{- if eq .Values.trivy.mode "ClientServer" }}
+  trivy.serverURL: {{ required ".Values.trivy.serverURL is required" .Values.trivy.serverURL | quote }}
+  {{- end }}
+  {{- with .Values.trivy.resources }}
+    {{- with .requests }}
+      {{- if .cpu }}
+  trivy.resources.requests.cpu: {{ .cpu }}
+      {{- end }}
+      {{- if hasKey . "memory" }}
+  trivy.resources.requests.memory: {{ .memory }}
+      {{- end }}
+    {{- end }}
+    {{- with .limits }}
+      {{- if .cpu }}
+  trivy.resources.limits.cpu: {{ .cpu }}
+      {{- end }}
+      {{- if .memory }}
+  trivy.resources.limits.memory: {{ .memory }}
+      {{- end }}
+    {{- end }}
+  {{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: starboard-trivy-config
+  labels:
+    {{- include "starboard-operator.labels" . | nindent 4 }}
+data:
+  {{- with .Values.trivy }}
+  {{- if .githubToken }}
+  trivy.githubToken: {{ .githubToken | b64enc | quote }}
+  {{- end }}
+  {{- if eq .mode "ClientServer" }}
+  {{- if .serverToken }}
+  trivy.serverToken: {{ .serverToken | b64enc | quote }}
+  {{- end }}
+  {{- if .serverCustomHeaders }}
+  trivy.serverCustomHeaders: {{ .serverCustomHeaders | b64enc | quote }}
   {{- end }}
   {{- end }}
-{{- if eq .Values.operator.configAuditReportsPlugin "Conftest" }}
+  {{- end }}
+{{- end }}
+{{- if eq .Values.starboard.configAuditReportsPlugin "Conftest" }}
 ---
 apiVersion: v1
 kind: ConfigMap
@@ -40,16 +92,45 @@ metadata:
   name: starboard-conftest-config
   labels:
     {{- include "starboard-operator.labels" . | nindent 4 }}
+data:
+  conftest.imageRef: {{ required ".Values.conftest.imageRef is required" .Values.conftest.imageRef | quote }}
+  conftest.resources.requests.cpu: {{ .Values.conftest.resources.requests.cpu | quote }}
+  conftest.resources.requests.memory: {{ .Values.conftest.resources.requests.memory | quote }}
+  conftest.resources.limits.cpu: {{ .Values.conftest.resources.limits.cpu | quote }}
+  conftest.resources.limits.memory: {{ .Values.conftest.resources.limits.memory | quote }}
 {{- end }}
-{{- if eq .Values.operator.configAuditReportsPlugin "Polaris" }}
+{{- if eq .Values.starboard.configAuditReportsPlugin "Polaris" }}
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: starboard-polaris-config
   labels:
-  {{- include "starboard-operator.labels" . | nindent 4 }}
+    {{- include "starboard-operator.labels" . | nindent 4 }}
 data:
+  polaris.imageRef: {{ required ".Values.polaris.imageRef is required" .Values.polaris.imageRef | quote }}
   polaris.config.yaml: |
   {{- toYaml .Values.polaris.config | nindent 4 }}
 {{- end }}
+{{- if eq .Values.starboard.vulnerabilityReportsPlugin "Aqua" }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: starboard-aqua-config
+  labels:
+     {{- include "starboard-operator.labels" . | nindent 4 }}
+data:
+  aqua.imageRef: {{ required ".Values.aqua.imageRef is required!" .Values.aqua.imageRef | quote }}
+  aqua.serverURL: {{ required ".Values.aqua.serverURL is required!" .Values.aqua.serverURL | quote }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: starboard-aqua-config
+  labels:
+    {{- include "starboard-operator.labels" . | nindent 4 }}
+data:
+  aqua.username: {{ required ".Values.aqua.username is required!" .Values.aqua.username | b64enc | quote }}
+  aqua.password: {{ required ".Values.aqua.password is required!" .Values.aqua.password | b64enc | quote }}
+{{- end}}

--- a/helmfile/upstream/starboard-operator/templates/rbac.yaml
+++ b/helmfile/upstream/starboard-operator/templates/rbac.yaml
@@ -33,6 +33,7 @@ rules:
       - pods
       - pods/log
       - replicationcontrollers
+      - services
     verbs:
       - get
       - list
@@ -40,7 +41,7 @@ rules:
   - apiGroups:
       - ""
     resources:
-      - "nodes"
+      - nodes
     verbs:
       - get
       - list
@@ -71,7 +72,7 @@ rules:
       - create
   - apiGroups:
       - apps
-    resources: # resources that own pods are inspected
+    resources:
       - replicasets
       - statefulsets
       - daemonsets
@@ -82,19 +83,36 @@ rules:
   - apiGroups:
       - batch
     resources:
+      - jobs
       - cronjobs
     verbs:
       - get
       - list
       - watch
   - apiGroups:
-      - batch
-    resources: # jobs are inspected and directly used by the starboard-operator
-      - jobs
+      - rbac.authorization.k8s.io
+    resources:
+      - roles
+      - rolebindings
+      - clusterroles
+      - clusterrolebindings
     verbs:
       - get
       - list
       - watch
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - batch
+    resources:
+      - jobs
+    verbs:
       - create
       - delete
   - apiGroups:
@@ -102,6 +120,7 @@ rules:
     resources:
       - vulnerabilityreports
       - configauditreports
+      - clusterconfigauditreports
       - ciskubebenchreports
     verbs:
       - get

--- a/helmfile/upstream/starboard-operator/values.yaml
+++ b/helmfile/upstream/starboard-operator/values.yaml
@@ -21,10 +21,6 @@ operator:
   configAuditScannerEnabled: true
   # kubernetesBenchmarkEnabled the flag to enable CIS Kubernetes Benchmark scanner
   kubernetesBenchmarkEnabled: true
-  # vulnerabilityReportsPlugin the name of the plugin that generates vulnerability reports. Either `Trivy` or `Aqua`.
-  vulnerabilityReportsPlugin: "Trivy"
-  # configAuditReportsPlugin the name of the plugin that generates config audit reports. Either `Polaris` or `Conftest`.
-  configAuditReportsPlugin: "Polaris"
   # batchDeleteLimit the maximum number of config audit reports deleted by the operator when the plugin's config has changed.
   batchDeleteLimit: 10
   # batchDeleteDelay the duration to wait before deleting another batch of config audit reports.
@@ -70,20 +66,99 @@ envSecret:
   stringData:
     OPERATOR_LOG_DEV_MODE: "false"
 
+starboard:
+  # vulnerabilityReportsPlugin the name of the plugin that generates vulnerability reports. Either `Trivy` or `Aqua`.
+  vulnerabilityReportsPlugin: "Trivy"
+  # configAuditReportsPlugin the name of the plugin that generates config audit reports. Either `Polaris` or `Conftest`.
+  configAuditReportsPlugin: "Polaris"
+
+  # scanJobTolerations tolerations to be applied to the scanner pods so that they can run on nodes with matching taints
+  scanJobTolerations: []
+  # If you do want to specify tolerations, uncomment the following lines, adjust them as necessary, and remove the
+  # square brackets after 'scanJobTolerations:'.
+  # - key: "key1"
+  #   operator: "Equal"
+  #   value: "value1"
+  #   effect: "NoSchedule"
+
+  # scanJobAnnotations comma-separated representation of the annotations which the user wants the scanner pods to be
+  # annotated with. Example: `foo=bar,env=stage` will annotate the scanner pods with the annotations `foo: bar` and `env: stage`
+  scanJobAnnotations: ""
+
 trivy:
-  imageRef: docker.io/aquasec/trivy:0.16.0
+  # imageRef the Trivy image reference.
+  imageRef: docker.io/aquasec/trivy:0.19.2
+
+  # mode is the Trivy client mode. Either Standalone or ClientServer. Depending
+  # on the active mode other settings might be applicable or required.
   mode: Standalone
-  serverURL:
-  httpProxy:
-  httpsProxy:
-  noProxy:
+
+  # httpProxy is the HTTP proxy used by Trivy to download the vulnerabilities database from GitHub.
+  #
+  # httpProxy:
+
+  # httpsProxy is the HTTPS proxy used by Trivy to download the vulnerabilities database from GitHub.
+  #
+  # httpsProxy:
+
+  # noProxy is a comma separated list of IPs and domain names that are not subject to proxy settings.
+  #
+  # noProxy:
+
+  # severity is a comma separated list of severity levels reported by Trivy.
   severity: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
 
+  # ignoreUnfixed is the flag to show only fixed vulnerabilities in
+  # vulnerabilities reported by Trivy. Set to "true" to enable it.
+  #
+  # ignoreUnfixed: "true"
+
+  # resources resource requests and limits
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100M
+    limits:
+      cpu: 500m
+      memory: 500M
+
+  # githubToken is the GitHub access token used by Trivy to download the vulnerabilities
+  # database from GitHub. Only applicable in Standalone mode.
+  #
+  # githubToken: "*****"
+
+  # serverURL is the endpoint URL of the Trivy server. Required in ClientServer mode.
+  #
+  # serverURL: "https://trivy.trivy:4975"
+
+  # serverToken is the token to authenticate Trivy client with Trivy server. Only
+  # applicable in ClientServer mode.
+  #
+  # serverToken: "*****"
+
+  # serverTokenHeader is the name of the HTTP header used to send the authentication
+  # token to Trivy server. Only application in ClientServer mode when
+  # trivy.serverToken is specified.
+  serverTokenHeader: "Trivy-Token"
+
+  # serverCustomHeaders is a comma separated list of custom HTTP headers sent by
+  # Trivy client to Trivy server. Only applicable in ClientServer mode.
+  #
+  # serverCustomHeaders: "foo=bar"
+
 kubeBench:
-  imageRef: docker.io/aquasec/kube-bench:0.5.0
+  imageRef: docker.io/aquasec/kube-bench:0.6.3
 
 polaris:
-  imageRef: quay.io/fairwinds/polaris:3.2
+  imageRef: quay.io/fairwinds/polaris:4.0
+  # resources resource requests and limits
+  resources:
+    requests:
+      cpu: 50m
+      memory: 50M
+    limits:
+      cpu: 300m
+      memory: 300M
   config:
     checks:
       # reliability
@@ -271,7 +346,26 @@ polaris:
 
 conftest:
   # imageRef the image reference
-  imageRef: docker.io/openpolicyagent/conftest:v0.23.0
+  imageRef: docker.io/openpolicyagent/conftest:v0.25.0
+  # resources resource requests and limits
+  resources:
+    requests:
+      cpu: 50m
+      memory: 50M
+    limits:
+      cpu: 300m
+      memory: 300M
+
+aqua:
+  # imageRef Aqua scanner image reference. The tag determines the version of the scanner binary executable and it must
+  # be compatible with version of Aqua server.
+  imageRef: docker.io/aquasec/scanner:5.3
+  # serverURL the endpoint URL of Aqua management console
+  serverURL:
+  # username the Aqua management console username
+  username:
+  # password the Aqua management console password
+  password:
 
 rbac:
   create: true

